tin/apps/courses/views.py

```diff
@@ -21,9 +21,15 @@
 @login_required
 def index_view(request):
     """Lists all courses"""
-    courses = Course.objects.filter_visible(request.user).order_by("-created")
+    courses = (
+        Course.objects.filter_visible(request.user).filter(archived=False).order_by("-created")
+    )
+    courses_expired = (
+        Course.objects.filter(archived=True).filter_permission(request.user, "r").distinct()
+    )
+    courses_expired = courses_expired.order_by("-created")
 
-    context = {"courses": courses}
+    context = {"courses": courses, "expired_courses": courses_expired}
```

Comment:
Is there a reason for the name switch from `archived` to `expired`?

Reply:
Nope, that was just a relic name from my previous revisions.